deploy/operator/ztp/deploy_spoke_cluster.sh

```diff
@@ -67,6 +67,12 @@ export -f wait_for_object_amount
 timeout 20m bash -c "wait_for_object_amount agent ${SPOKE_CONTROLPLANE_AGENTS} 30 ${SPOKE_NAMESPACE}"
 echo "All ${SPOKE_CONTROLPLANE_AGENTS} agents have been discovered!"
 
+# Only set by CI. Should not be used for local deployment/testing.
+if [[ "${WAIT_FOR_ASSISTED_FULL_INSTALL_ZTP}" == "false" ]]; then
```

Comment:
I commented in https://github.com/openshift/release/pull/21569 with regards to the variable name